infrastructure/docker/docker-compose.yml

```diff
@@ -12,6 +12,7 @@ services:
             - "8080:8080"
         volumes:
             - "/var/run/docker.sock:/var/run/docker.sock"
+        network_mode: "host"
```

Comment:
I'm not sure exposing the router to local machine is a good thing

Reply:
Why? I was already the case before that :)

Reply:
indeed... I forgot about port mapping :)